ui/page/livestreamSetup/view.jsx

```diff
@@ -192,24 +184,6 @@ export default function LivestreamSetupPage(props: Props) {
         </>
       )}
 
-      {/* no livestreaming privs because no premium membership */}
-      {!livestreamEnabled && !odyseeMembership && (
-        <div style={{ marginTop: '11px' }}>
-          <h2 style={{ marginBottom: '15px' }}>
-            {__('To stream on Odysee, please join Odysee Premium or have 50 Credits as support on your channel')}
-          </h2>
-
-          <Button
-            button="primary"
-            label={__('Join Odysee Premium')}
-            icon={ICONS.FINANCE}
-            navigate={`/$/${PAGES.ODYSEE_MEMBERSHIP}`}
-            className="membership_button"
-            style={{ maxWidth: '238px' }}
-          />
-        </div>
-      )}
```

Comment:
Page becomes empty (with only channel selector) if the livestream is disabled.
But probably minor since if a user demanded a "disabled" tag, they should know what's going on.

Reply:
Good point, only scammers are disabled, but I'll add this back.